example/spec/parallel/ingestGranule/IngestGranuleFailureSpec.js

```diff
@@ -166,6 +169,12 @@ describe('The Ingest Granule failure workflow', () => {
     });
 
     it('fails the granule with the error message', async () => {
+      await waitForModelStatus(
+        granuleModel,
+        { granuleId: inputPayload.granules[0].granuleId },
+        'completed'
```

Comment:
You're expecting the granule status to be failed on line 184, so you'll want to change this to `'failed'`.